src/mscorlib/src/System/Threading/Thread.cs

```diff
@@ -122,7 +124,7 @@ internal ThreadHandle(IntPtr pThread)
     [ClassInterface(ClassInterfaceType.None)]
     [ComDefaultInterface(typeof(_Thread))]
 [System.Runtime.InteropServices.ComVisible(true)]
-    public sealed class Thread : CriticalFinalizerObject, _Thread
+    public sealed class Thread : CriticalFinalizerObject, _Thread, IRuntimeThread
```

Comment:
It would be more efficient to make this a base class instead of interface (interface calls are slower, and we can easily avoid the virtual calls completely for CoreRT):

```
class RuntimeThread : CriticalFinalizerObject
{
...
}

class Thread : RuntimeThread, _Thread
{
...
}
```